core/src/main/java/com/amazon/opendistroforelasticsearch/sql/expression/operator/arthmetic/MathematicalFunction.java

```diff
@@ -97,6 +110,58 @@ private static FunctionResolver ceiling() {
             .build());
   }
 
+  /**
+   * Definition of conv(x, a, b) function.
+   * Convert number x from base a to base b
+   * The supported signature of floor function is
+   * (STRING, INTEGER, INTEGER) -> STRING
+   */
+  private static FunctionResolver conv() {
+    FunctionName functionName = BuiltinFunctionName.CONV.getName();
+    return new FunctionResolver(
+        functionName,
+        new ImmutableMap.Builder<FunctionSignature, FunctionBuilder>()
+            .put(
+                new FunctionSignature(functionName,
+                    Arrays.asList(ExprCoreType.STRING, ExprCoreType.INTEGER, ExprCoreType.INTEGER)),
+                tripleArgFunc(
+                    functionName, (t, u, v) -> Integer.toString(Integer.parseInt(t, u), v),
```

Comment:
Could we use more readable argument name such as `(num, fromBase, toBase) ->` ?

Reply:
sounds good, will do